docs/docs/using-graphql-playground.md

```diff
@@ -4,20 +4,22 @@ title: Using the GraphQL Playground
 
 ## Intro
 
-In this page we want to introduce you to an alternative to the current IDE for your GraphQL queries: The [Prisma Playground](https://github.com/prisma/graphql-playground).
+In this page we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
 
-## What is The Prisma Playground?
+## What is Prisma's GraphQL Playground?
 
-The Playground is a way for you to interact with the data your sources and plugins add as schemas. You will be interacting with this data a lot and the Playground will help you greatly with exploring this data.
+GraphQL Playground is a way for you to interact with the data your sources and plugins add as schemas. You will be interacting with this data a lot and the Playground will help you greatly with exploring this data.
 
 ## Accessing the Playground
 
-As this is an experimental feature, you need to access it by adding an environmental variable, you need to add `GATSBY_GRAPHQL_IDE` to your `develop` script in your `package.json`, like this:
+To access GraphQL Playground add `GATSBY_GRAPHQL_IDE` to your `develop` script in your `package.json`, like this:
```

Comment:
Could you add back the note about this being an experimental feature? That's important as we may remove it in a future update so people shouldn't expect this to stay around.

Reply:
Sent a new commit with the note included again!